### PR TITLE
Pluginized a soundJs plugin

### PR DESCRIPTION
**Pull request by brianpabent**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ All plugins in this repository are licensed under the MIT license. A copy of the
 This plugin enables tracking of page views within Ceros experiences in Oracle Eloqua. On each page change, the URL of the page
 will be tracked under the configured Eloqua Site ID.
 
-To use this plugin, paste the following code into the "Custom HTML" field of the Ceros experience inside Ceros Studio. You can
+### How to use this plugin: 
+
+paste the following code into the "Custom HTML" field of the Ceros experience inside Ceros Studio. You can
```
Comment (addelong) [inline README.md]: please capitalize

File: README.md [hunk 2]
```diff
@@ -20,4 +22,20 @@ icon in the upper right of Eloqua, clicking "Setup", and then clicking "Company
...
+## SoundJs Plugin
+
+Uses the [SoundJs](http://www.createjs.com/soundjs) library to enable people using the Ceros Studio to create an experience that will play a sound when
+an object is clicked.
+
+The sound file must be hosted on a server that allows [cross origin requests](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing)
+
+### How to use this plugin:
+
+1. paste the following code into the "Custom HTML" field of the Ceros experience inside Ceros Studio.
```
Comment (addelong) [inline README.md]: ditto

File: src/soundjs.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+(function() {
+
+    if (typeof(CerosSDK) === "undefined") {
+        var sdkScript = document.createElement('script');
+        sdkScript.type = "text/javascript";
+        sdkScript.async = true;
+        sdkScript.onload = activatePlaySound;
+        sdkScript.src = "//sdk.ceros.com/standalone-player-sdk-v3.js";
+
+        document.getElementsByTagName('head')[0].appendChild(sdkScript);
```
Comment (addelong) [inline src/soundjs.js]: Need to `activatePlaySound` here in the if block too, I think.

Comment (addelong) [inline src/soundjs.js]: Or actually I think just get rid of the else block and always `activatePlaySound()`.

File: src/soundjs.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    function activatePlaySound() {
+        loadSoundJs();
+
+        CerosSDK.findExperience().done(function(ceros) {
+            var pluginScriptTag = document.getElementById("ceros-soundjs-plugin");
+            var soundTag = pluginScriptTag.getAttribute("soundTag");
+            var componentsWithSound = ceros.findComponentsByTag(soundTag);
+            jQuery.each(componentsWithSound.components, function (soundComponentIndex, soundComponent) {
```
Comment (addelong) [inline src/soundjs.js]: Where's jQuery come from?

File: src/soundjs.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        soundScript.type = "text/javascript";
+        soundScript.async = true;
+        soundScript.src = "//code.createjs.com/soundjs-0.6.2.min.js";
+        document.body.appendChild(sdkScript);
+    }
+
+    function activatePlaySound() {
+        loadSoundJs();
+
+        CerosSDK.findExperience().done(function(ceros) {
```
Comment (addelong) [inline src/soundjs.js]: Can we rename `ceros` to `experience`? Since that is what it is.

File: Gruntfile.js [hunk 6]
```diff
@@ -73,17 +79,15 @@ module.exports = function(grunt) {
                     out: './dist/plugins/eloqua/main.js',
                     skipSemiColonInsertion: true
                 }
-
-            }
-        },
-        compress: {
-            release: {
+            },
+            compile_eloqua_plugin: {
```
Comment (addelong) [inline Gruntfile.js]: Can't have duplicate `compile_eloqua_plugin` declarations here.

File: src/soundjs.js (hunk 3 above)
Comment (brianpabent) [inline src/soundjs.js]: `activatePlaySound()` is called from sdkScript.onload. I copied this pattern from the Eloqua plugin.

File: src/soundjs.js (hunk 4 above)
Comment (brianpabent) [inline src/soundjs.js]: jQuery is assumed to be available in Ceros Player

File: src/soundjs.js (hunk 5 above)
Comment (brianpabent) [inline src/soundjs.js]: yes. good call.

**Comment by brianpabent** [comment]
@addelong - this is ready for another review.
@grahamedaley - this can be tested.

**Comment by brianpabent** [comment]
@grahamedaley - here is a studio with SDK tags set up:
https://admin.ceros.com/account/ceros-qa/studio/experience/issue-56f43e6883631

**Comment by addelong** [comment]
Code 👍 🔉 🔊 🔇

**Comment by grahamedaley** [comment]
Functional 👍 🔈

**Pull request closed without merging**